docs/asciidoc/modules/ROOT/pages/ml/genai.adoc

```diff
@@ -102,6 +103,104 @@ RETURN m.title
 |===
 
 
+We can use the `additionalPrompts` config to improve the request, e.g. adding the output of the `apoc.ml.schema`.
```

Comment:
We can use the `additionalPrompts` config to improve the request, e.g. adding the natural language description of the schema ( like the output of the `apoc.ml.schema` for instance).